ci/jenkins/jobs/macros.yaml

```diff
@@ -350,6 +398,68 @@
           fi
           echo "=== TEST SUCCESS !!! ==="
 
+- builder:
+    name: builder-e2e-coverage
+    builders:
+      - shell: |-
+          #!/bin/bash
+          set -ex
+          echo ====== Running Antrea E2E Tests ======
+
+          export GO111MODULE=on
+          export WORK_HOME=/var/lib/jenkins
+          export GOPATH=$WORK_HOME/go
+          export GOROOT=/usr/local/go
+          export GOCACHE=$WORK_HOME/.cache/go-build
+          export PATH=$GOROOT/bin:$PATH
+          export KUBECONFIG=${WORKSPACE}/jenkins/out/kubeconfig
+          cluster="${JOB_NAME}-${BUILD_NUMBER}"
+
+          mkdir -p test/e2e/infra/vagrant/playbook/kube
+          cp -f "${WORKSPACE}/jenkins/out/kubeconfig" test/e2e/infra/vagrant/playbook/kube/config
+
+          echo "=== Generate ssh-config ==="
+          cp -f ci/jenkins/ssh-config test/e2e/infra/vagrant/ssh-config
+          master_name="$(kubectl get nodes -o wide --no-headers=true | awk '$3 == "master" {print $1}')"
+          master_ip="""$(kubectl get nodes -o wide --no-headers=true | awk '$3 == "master" {print $6}')"
+          echo "=== Master node ip: ${master_ip} ==="
+          sed -i "s/MASTERNODEIP/${master_ip}/g" test/e2e/infra/vagrant/ssh-config
+          echo "=== Move kubeconfig to master ==="
+          ssh -o StrictHostKeyChecking=no -i jenkins/key/antrea-ci-key -n capv@${master_ip} "mkdir .kube"
+          scp -o StrictHostKeyChecking=no -i jenkins/key/antrea-ci-key jenkins/out/kubeconfig capv@${master_ip}:~/.kube/config
+          sed -i "s/CONTROLPLANENODE/${master_name}/g" test/e2e/infra/vagrant/ssh-config
+          echo "    IdentityFile ${WORKSPACE}/jenkins/key/antrea-ci-key" >> test/e2e/infra/vagrant/ssh-config
+
+          # Run and configure Prometheus
+          kubectl apply -f build/yamls/antrea-prometheus.yml
+
+          set +e
+          mkdir -p `pwd`/antrea-test-logs
+          mkdir -p `pwd`/e2e-coverage
+          go test -v -timeout=50m github.com/vmware-tanzu/antrea/test/e2e --logs-export-dir `pwd`/antrea-test-logs --prometheus --coverage --coverage-dir `pwd`/e2e-coverage
```

Comment:
There seems to be little change from `builder-e2e`. Could you reuse code? You can refer to `builder-conformance`.

Reply:
done, reuse code

Reply:
done

Reply:
done